autogen/main/cluster.tf.tmpl

```diff
@@ -847,6 +847,13 @@ resource "google_container_node_pool" "windows_pools" {
         local_ssd_count = ephemeral_storage_config.value
       }
     }
+
```

Comment:
We should pull the `dynamic "local_nvme_ssd_block_config"` out of the `if beta_cluster` block

```suggestion
{% endif %}

```